source/extensions/filters/http/dynamo/dynamo_request_parser.cc

```diff
@@ -39,6 +39,7 @@ const std::vector<std::string> RequestParser::SUPPORTED_ERROR_TYPES{
     "ConditionalCheckFailedException",
     "IdempotentParameterMismatchException",
     "IncompleteSignatureException",
+    "InvalidSignatureException",
```

Comment:
While you at this, could you add a link to dynamo docs providing the list of supported error types?

Reply:
@dio Thanks for reviewing the PR :) 
This error is not listed in the [error handling section](https://docs.aws.amazon.com/amazondynamodb/latest/developerguide/Programming.Errors.html#Programming.Errors.MessagesAndCodes) of DynamoDB developer guide, but users do get this error for [various reasons](https://github.com/aws/aws-sdk-go/issues/2598#issuecomment-526398896). What is the recommended way of adding this info into the repo?

Reply:
Exactly what you have said above will be good as inline comments. Telling that these are the errors listed in the docs and there is this one observed at runtime. Thank you!

Reply:
Gotcha. I added the info as inline comments. (plus fixing the obsolete url link)